.travis.yml

```diff
@@ -4,4 +4,7 @@ script:
 language: node_js
 
 node_js:
-  - 0.10
+  - "0.10"
+  - "iojs"
```

Comment:
Hmm, I want to test on only minimal environments.
It make CI fast.

So, I think we have 2 ways:

- Test on v0.10 (legacy env) and iojs (latest env)
- Test on v0.12 and iojs

I think v0.10 is better. Because if tests pass on v0.10, the tests almost pass on v0.12.
What do you think?